### PR TITLE
git-extras 4.6.0

### PR DESCRIPTION
**Pull request by spacewander**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This pr upgrades git-extras from 4.5.0 to 4.6.0.

### CODE REVIEW

File: Formula/git-extras.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GitExtras < Formula
   desc "Small git utilities"
   homepage "https://github.com/tj/git-extras"
-  url "https://github.com/tj/git-extras/archive/4.5.0.tar.gz"
-  sha256 "cb099d9e155c3bf863f95dd91c72bcc2e05fb28e3ebce527cd70d1b517402615"
+  url "https://github.com/tj/git-extras/archive/4.6.0.tar.gz"
+  sha256 "e3baa3a54cb3d8f2e09409e9dec22c76c0e4802e44e108b6c"
```
Comment (DomT4) [inline Formula/git-extras.rb]: This is a `sha1` rather than `sha256` checksum. We want the latter, please!

Comment (spacewander) [reply Formula/git-extras.rb]: @DomT4 
My bad! Fixed in the new commit.

**Comment by spacewander** [comment]
@DomT4 
Thank you for your review. I just push another commit to fix the problem. Would you review it again?

**Comment by DomT4** [comment]
Thanks for the quick response! It'll automatically go back through our CI for another check. Shouldn't take too long, hopefully.

**Pull request closed without merging**